packages/api/src/routes/interaction.ts

```diff
@@ -149,13 +154,125 @@ const interactions: FastifyPluginAsync = async (fastify): Promise<void> => {
           to: toPlayer.username,
           points,
           timestamp: currentTimestamp,
+          socialsFrom: fromPlayer.socials?.share
+            ? validateSocials(fromPlayer.socials)
+            : null,
+          socialsTo: toPlayer.socials?.share
+            ? validateSocials(toPlayer.socials)
+            : null,
         })
-
         return reply.status(200).send(interaction)
       },
     }
   )
+  fastify.post<{ Body: ShareSocialsParams; Reply: ShareSocialsResult | Error }>(
+    '/share',
```

Comment:
REST APIs are focus on `resources` and the common way to define routes is using nouns:
- `/resource1/resource2/`

```suggestion
    '/player/socials',
    
    '/socials/'
```